src/index.js

```diff
@@ -0,0 +1,5 @@
+const app = require("./app");
+
+app.listen(app.get("PORT"), () => {
+  console.log("The magic is happening on port", app.get("PORT"));
```

Comment:
put the link of server
```suggestion
  console.log(`the server is running at http://localhost:${app.get('PORT')}`);
```